scripts/non_visual_qa.py

```diff
@@ -152,7 +152,7 @@ def non_visual_qa(files: List[str]) -> None:
     get_log().info("non_visual_qa_end", source=files, duration=time_in_ms() - start_time)
 
 
-def main() -> None:
+def main() -> None:  # pylint: disable=duplicate-code
```

Comment:
That's annoying that pylint does not raise this warning locally via `pre-commit` script